pages/my-collections.tsx

```diff
@@ -0,0 +1,122 @@
+import Image from 'next/image';
+import withTransition from '../components/withTransition';
+import { ConnectToWalletBanner } from '../components/ConnectToWalletBanner';
+import { Loader, Banner } from '../components';
+import { addressToCreatorImage, shortenAddress } from '../utils/address';
+import { useAlephiumConnectContext } from '@alephium/web3-react';
+import { useEffect, useState } from 'react';
+import axios from 'axios';
+import Link from 'next/link';
+import { motion } from 'framer-motion';
+import { shortenName } from '../utils';
+
+interface NFTCollection {
+  _id: string  // NFTCollection contract id
+  type: 'NFTOpenCollection' | 'NFTPublicSaleCollection'
```

Comment:
Seems that we are still setting `NFTPublicSaleCollectionSequential` in the code.

Reply:
I have cherry-picked it from the old branch, will fix it, thanks!